Class-Code/OOP/blehrer/sketch.js

```diff
@@ -1,14 +1,58 @@
-//Your Name(s)
-//Project Name
-//Date
+//Ben Lehrer
+//In Class Assignment
+//4/9/19
 
-/* A brief description of what the program is */
+/*
+    In your fork of cs2-student-files navigate to OOP/object-classes/yourname.
+    (If it’s not there, make sure to pull in the updates from fcs-cs to your fork)
+    There you will find a sketch of a bouncing ball written without using object classes.
+    Refactor the code so that there is a Ball class that knows about its x, y and r
+    and has at least three methods. After refactoring create at least two instances
+    of the Ball class (ball1 and ball2). When you have finished submit a pull request
+    back to fcs-cs/cs2-student-files MASTER.
+*/
 
+class Bubble {
+    constructor(_x, _y, _r) {
```

Comment:
I see this often. Is this an Arduino or C convention? I like it because I think it's more clear that they're temp variables that are getting passed into the object.